src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -211,16 +188,18 @@ private async Task UpdateJobRecordAsync(CancellationToken cancellationToken)
             }
         }
 
-        // Get destination info from secret store, create appropriate export client and connect to destination.
-        private async Task GetDestinationInfoAndConnectAsync(CancellationToken cancellationToken)
+        // Create appropriate export client and connect to destination.
+        private async Task GetDestinationClientAndConnectAsync(CancellationToken cancellationToken)
```

Comment:
This still doesn't address the issue with expiring access token (e.g., it's acquiring token before the while loop but the while loop could take longer than the validation period of the access token). Would that be fixed in this PR or another PR?

Reply:
Yes. I will add that ability along with retries as part of another PR. The corresponding WI is #72260